projects/Mallard/src/authentication/__tests__/credentials-chain.spec.ts

```diff
@@ -219,6 +223,41 @@ describe('credentials-chain', () => {
                 casDataCache,
                 legacyCasExpiryCache,
                 iapReceiptCache,
+                false,
+            )
+
+            expect(result).toEqual(unauthed)
+        })
+
+        it('returns an `iap` when we have only an expired iap receipt and are in testflight', async () => {
+            const userDataCache = getMockAsyncCache(null)
+            const casDataCache = getMockAsyncCache(invalidCasExpiry)
+            const legacyCasExpiryCache = getMockCache(invalidCasExpiry)
```

Comment:
Should this and the one above both be `invalidCasExpiry `?

Reply:
Yeah, they're both CAS caches, one is for the legacy CAS from the old app, the other is for our cache.